zh/publish/publish-native.md

```diff
@@ -12,6 +12,8 @@
 
 将图集中的全部 SpriteFrame 合并到同一个包中。默认关闭，启用后能够减少热更新时需要下载的 SpriteFrame 文件数量，但如果图集中的 SpriteFrame 数量很多，则可能会延长原生平台上的启动时间。
 
+**注意**：实现热更新项目功能时，新旧项目需要同步使用 **合并图集中的 SpriteFrame** 功能，否则会导致热更之后资源引用错误的问题。
```

Comment:
建议修改：
**注意**：在热更新时，需要确保新旧项目中该功能的开启/关闭状态保持一致，否则会导致热更新之后出现资源引用错误的情况。

还有麻烦帮忙英文版也同步修改一下：
**Note**: For projects using hot update, please ensure that the status of this option is consistent during version upgrades, otherwise it will cause resource error in new version.